src/Rules/Library/SiblingUniqueAndNotFocusable.cs

```diff
@@ -39,8 +39,8 @@ public SiblingUniqueAndNotFocusable()
 
         public override EvaluationCode Evaluate(IA11yElement e)
         {
-            if (e == null) throw new ArgumentException(nameof(e));
-            if (e.Parent == null) throw new ArgumentNullException(nameof(e.Parent));
+            if (e == null) throw new ArgumentNullException(nameof(e));
+            if (e == null) throw new ArgumentNullException(nameof(e));
```

Comment:
Lines 42 and 43 are identical. I suspect line 43 is supposed to be
```suggestion
            if (e.Parent == null) throw new ArgumentException(ErrorMessages.ElementParentNull, nameof(e));
```